api/id/id.go

```diff
@@ -63,13 +63,13 @@ func New() Id {
 
 	id[8] = byte(machineID<<4) | byte((count<<4)>>60)
```

Comment:
think i had an epic brain fart when i changed the machine id to 48 bits. this looks to use 20 bits and also be messed up. not sure how i get paid for this